docs/issue_template.md

```diff
@@ -0,0 +1,15 @@
+### Difficulty: M
+Based on ***complexity***, ranging from **XS - XL** (👕 sizes)
+
+### Suggested Deadline: 1 week
+
+---
+
+## User Story
+As a user, I want to ...
```

Comment:
Found [this template](https://www.mountaingoatsoftware.com/agile/user-stories) for a user story:

> As a < type of user >, I want < some goal > so that < some reason >.

I think it might be helpful to include _goal_ and _reason_ in the template. Lmk what you think.

Reply:
Yes! thanks